src/models/bulletChart.js

```diff
@@ -189,6 +194,7 @@ nv.models.bulletChart = function() {
         tickFormat:    {get: function(){return tickFormat;}, set: function(_){tickFormat=_;}},
         ticks:    {get: function(){return ticks;}, set: function(_){ticks=_;}},
         noData:    {get: function(){return noData;}, set: function(_){noData=_;}},
+        axisDuration:    {get: function(){return axisDuration;}, set: function(_){axisDuration=_;}},
```

Comment:
This appears to be a duration just for the axis, so this name seems fine, but please document any new options you add in the json at the bottom of documentation.html